widgets/widget-featured-page.php

```diff
@@ -196,6 +198,17 @@ public function form( $instance ) {
 					if ( 'inline' === '<?php echo esc_attr( $layout ); ?>' ) {
 						$( '.js-featured-page-settings__additional-block-settings' ).hide();
 					}
+
+					$( "#<?php echo esc_attr( $this->get_field_id( 'page_id' ) ); ?>" )
+						.change( function(){
+							var selection = $( "#<?php echo esc_attr( $this->get_field_id( 'page_id' ) ); ?> option:selected" );
+							var button = $( "#<?php echo esc_attr( $this->get_field_id( 'page_id' ) ); ?>-edit-link" );
+							var href = '<?php echo admin_url(); ?>/post.php?post=' + selection.val() + '&action=edit';
```

Comment:
The URL has a double slash (/). Remove the slash character after the admin_url call.